Objects/genericaliasobject.c

```diff
@@ -604,8 +607,13 @@ static PyObject *
 ga_reduce(PyObject *self, PyObject *Py_UNUSED(ignored))
 {
     gaobject *alias = (gaobject *)self;
-    return Py_BuildValue("O(OO)", Py_TYPE(alias),
-                         alias->origin, alias->args);
+    PyObject *starred = PyBool_FromLong(alias->starred);
+    PyObject *value = Py_BuildValue("O(OOO)", Py_TYPE(alias),
+                                    alias->origin, alias->args, starred);
+    // Avoid double increment of reference count on Py_True/Py_False - once from
+    // PyBool_FromLong, and once from Py_BuildValue.
+    Py_CLEAR(starred);
+    return value;
```

Comment:
How about:
```suggestion
    PyObject *starred = alias->starred ? Py_True : Py_False;
    return Py_BuildValue("O(OOO)", Py_TYPE(alias),
                                    alias->origin, alias->args, starred);

```

BTW, if you don't want to use the code above, you can avoid the double incref by using the format string ``O(OON)``. ``N`` doesn't incref. https://docs.python.org/3/c-api/arg.html#c.Py_BuildValue